unlock-app/src/components/interface/locks/CheckoutUrl/ChooseConfiguration.tsx

```diff
@@ -89,56 +98,149 @@ export function ChooseConfiguration({
   const [selectedConfig, setSelectedConfig] = useState<
     CheckoutConfig | undefined
   >(undefined)
+  const [customConfigId, setCustomConfigId] = useState<string | null>(null)
+  const [isCheckingPermissions, setIsCheckingPermissions] = useState(false)
+  const [locksToCheck, setLocksToCheck] = useState<
+    Array<{ address: string; network: number }>
+  >([])
 
-  const [defaultValue, setDefaultValue] = useState<string | null>(
-    selectedConfig?.id || value?.id
+  const { account } = useAuth()
+
+  // fetch the configuratoin's details when customConfigId changes
+  const { data: configDetails, isLoading: isLoadingConfig } = useCheckoutConfig(
+    { id: customConfigId! }
   )
 
+  // check if user is a manager for the current lock
+  const { isManager, isPending: isCheckingLockManager } = useLockManager({
+    lockAddress: locksToCheck[0]?.address,
+    network: locksToCheck[0]?.network,
+  })
```

Comment:
Shouldn't we check for _any_ lock in the config?

Reply:
it does, in the other `useEffect`